runners/console/index.ts

```diff
@@ -254,6 +254,30 @@ export class Console extends Runner {
         return result;
     }
 
+    runDownloadFile(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let downloadUrl = command.parameters[0];
+        let installDir = this.getVariable(this.workspaceDirectory);
+        if (command.parameters.length == 2) {
+            installDir = path.join(installDir, command.parameters[1]);
+            this.createFolder(installDir, false);
+        }
+        var command1;
+        var command2;
+        if(this.platform == ConsolePlatform.WINDOWS) {
+            command1 = "powershell.exe \"Invoke-WebRequest -OutFile download.tar.gz '" + downloadUrl + "'\"";
+            command2 = "powershell.exe tar -xvzf download.tar.gz";
```

Comment:
The file can have any file format, so downloading it as ".tar.gz" would cause problems. I think we also need to pass the filename as a parameter, and then use that name.
Unzipping a zipped file is not part of this command. I think this should be implemented separately